lib/plugins/install/tests/install.js

```diff
@@ -77,5 +129,18 @@ describe('Install', () => {
         expect(downloadStub.args[0][0]).to.equal(`${install.options.url}/archive/master.zip`);
       });
     });
+
+    it('should download and rename the service based on the GitHub URL', () => {
```

Comment:
Now the downloadStub mocks the zip loading and saves serverless.yml, then installation process renames the service value and finally the result is tested.